modules/imaging_browser/php/imagingbrowserrowprovisioner.class.inc

```diff
@@ -189,24 +193,24 @@ class ImagingBrowserRowProvisioner extends \LORIS\Data\Provisioners\DBRowProvisi
               s.ID as sessionID,
               GROUP_CONCAT(DISTINCT modality.Scan_type) as sequenceType,
               $PendingNewquery as pending," .
-              ($modalities_subquery==''?'':"$modalities_subquery,") .
-              "s.CenterID as CenterID,
+            ($modalities_subquery==''?'':"$modalities_subquery,") .
+            "s.CenterID as CenterID,
               c.Entity_type as entityType,
               s.ProjectID
-            FROM psc AS p 
-              JOIN session s ON (s.CenterID=p.CenterID) 
-              JOIN candidate c ON (c.CandID=s.CandID) 
+            FROM psc AS p
+              JOIN session s ON (s.CenterID=p.CenterID)
+              JOIN candidate c ON (c.CandID=s.CandID)
               LEFT JOIN Project ON (s.ProjectID=Project.ProjectID)
-              JOIN files f ON (f.SessionID=s.ID) 
-              LEFT JOIN files_qcstatus fqc ON (fqc.FileID=f.FileID) 
+              JOIN files f ON (f.SessionID=s.ID)
+              LEFT JOIN files_qcstatus fqc ON (fqc.FileID=f.FileID)
               JOIN parameter_file pf ON (f.FileID=pf.FileID)
               JOIN parameter_type pt USING (ParameterTypeID)
-              LEFT JOIN mri_scan_type modality ON 
+              LEFT JOIN mri_scan_type modality ON
                 (f.AcquisitionProtocolID=modality.ID)
-              $left_joins 
-            WHERE 
+              $left_joins
+            WHERE
               s.Active = 'Y' AND
-              f.FileType IN ('mnc', 'nii')
+              f.FileType='mnc'
```

Comment:
This is a regression of #6593

Reply:
Sorry, I don't know how this slipped, I amended the latest commit to change that line to how it was.